.pre-commit-config.yaml

```diff
@@ -1,11 +0,0 @@
-repos:
-- repo: https://github.com/softwaremill/scala-pre-commit-hooks
-  rev: v0.3.0
-  hooks:
-  - id: sbt-scalafmt
-    stages: [commit]
-- repo: https://github.com/pre-commit/pre-commit-hooks
-  rev: v4.3.0
-  hooks:
-  - id: end-of-file-fixer
-  - id: trailing-whitespace
```

Comment:
I'm not sure I agree with this change. I prefer for every commit to be formatted properly, rather than just at the time of the PR. Technically speaking though, you don't need to install or run pre-commit locally. It just exists for people who want to make sure their commits are formatted properly before committing (i.e. me).

This also matches with the settings I have for other repos, which makes it easier for me to make cross-cutting changes to the workflow and formatting of these projects.

Is there anything in this PR that would conflict with keeping these pre-commit-hooks?

Reply:
I have reverted this change